runtime/runtime-params-estimator/src/testbed_runners.rs

```diff
@@ -194,22 +195,22 @@ where
         }
     };
 
-    let bar = ProgressBar::new(warmup_total_transactions(config) as _);
-    bar.set_style(ProgressStyle::default_bar().template(
+    let progress_bar = ProgressBar::new(warmup_total_transactions(config) as _);
```

Comment:
https://rust-lang.github.io/rust-clippy/master/index.html#blacklisted_name

`clippy` has a check to avoid temporary names like "foo" and "bar". Obviously in this case the name `bar` actually makes sense, but it didn't seem like a big deal to me to rename the variable to be slightly more explicit. Of course, we could always ignore this lint if you disagree.